.github/workflows/CompatHelper.yml

```
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: julia-actions/setup-julia@latest
         with:
-          version: "1.3"
+          version: "1"
```

```suggestion
          version: "1.6"
```

We don't want CompatHelper suggesting changes that aren't compatible with the minimum Julia version (AFAICT from various GitHub and Slack conversations, the resolution algorithm is currently tied to the running version of Julia)